swiggy-pdp-cart/src/components/Main/Content/Categories/Categories.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import CategoryCSS from './categories.module.css'
+export default function Categories({ categoryList, category, setCategory }) {
+  return (
+    <div className={CategoryCSS.categories + ' col-4'}>
+      <ul>
+        {categoryList.map((item) => {
+          const { id, displayName } = item;
+          return (
+            <li
+              onClick={(e) => setCategory(e.target.id)}
+              className={id === category ? CategoryCSS.highlighted : ""}
```

Comment:
Instead of doing this inline, compute this on top and then pass it here. This will improve the readability of your code